tests/handlers/test_appservice.py

```diff
@@ -325,17 +324,19 @@ def test_notify_interested_services_ephemeral_out_of_order(self):
             interested_service, ephemeral=[]
         )
 
-    def _mkservice(self, is_interested, protocols=None):
+    def _mkservice(self, is_interested: bool, protocols=None):
```

Comment:
Should we update this parameter name too while we're here?

Reply:
The functionality of this `is_interested` parameter has actually changed from just tweaking whether the created appservice mock will be interested in events, to any event, user or room.

I've fixed this, as well as added docstrings and types in eb779e030.